signer/src/storage/model.rs

```diff
@@ -542,17 +542,18 @@ pub struct EncryptedDkgShares {
 /// Persisted public DKG shares from other signers
 #[derive(Debug, Clone, Hash, PartialEq, Eq, PartialOrd, Ord, sqlx::FromRow)]
 #[cfg_attr(feature = "testing", derive(fake::Dummy))]
-pub struct RotateKeysTransaction {
+pub struct KeyRotationEvent {
     /// Transaction ID.
     pub txid: StacksTxId,
+    /// The Stacks block ID of the block that includes the transaction
+    /// associated with this withdrawal request.
+    pub block_hash: StacksBlockHash,
     /// The address that deployed the contract.
     pub address: StacksPrincipal,
```

Comment:
nit: can we also update this one comment? IIUC it's the current principal authorized to operate on it, it's not related to the deployer (after the first multisig rotation)

Reply:
Closing as @matteojug is out of office and the nit is resolved in the linked commit.